### ISSUE TITLE
Unable to import tensorflow on colab

### ISSUE THREAD

**Issue by sujit25** [googlecolab/colabtools#868]
Bug report for Colab: http://colab.research.google.com/.

For questions about colab usage, please use [stackoverflow](https://stackoverflow.com/questions/tagged/google-colaboratory/).

- Describe the current behavior:
Getting error while importing tensorflow on colab. Failed to import libcublabs.so

- Describe the expected behavior:
After importing tensorflow, it should get successfully imported in notebook.

- The web browser you are using (Chrome, Firefox, Safari, etc.):
Chrome
- Link to self-contained notebook that reproduces this issue
  (click the *Share* button, then *Get Shareable Link*):
https://colab.research.google.com/drive/1qGCF1lO7n4qwvHZsSiZfXqE0G1zmyYD3
![colab_tensorflow_error](https://user-images.githubusercontent.com/8065697/69214383-d7ad3080-0b8c-11ea-9cc0-a088c3d4b7d6.PNG)

**Comment by sujit25** [comment]
This issue is observed only after installing tensorflow-gpu==2.0.0.alpha0 on runtime. If we proceed without installing it, everything works perfectly fine.

**Comment by colaboratory-team** [comment]
b/144844033

**Comment by colaboratory-team** [comment]
Duplicate of #864.
